docs/delete-response-codes.md

```diff
@@ -0,0 +1,27 @@
+# DeleteResponseCodes
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI (swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Delete-V1-01
+
+## Output Message
+
+Synchronous delete operations must have 200, 204, and default responses. They must not have any other responses.
```

Comment:
By reading the code I think you mean both.

Reply:
Yes, both. The rule ensures that the spec has all those responses defined, which covers the first point. We can't ensure the second point with a linter rule (that requires some sort of live validation) but that is the expectation.

Reply:
Sorry, I have another basic question: why we cannot validate with linter the second rule, but can the first? I mean, we could claim in the swagger spec that we return 200, 204 and so on, but in reality implement it in such a way that it only ever returns 200. Then we will break the first rule, and we won't be able to validate even with live validation, as it cannot can only tell us what it saw, but not what it will never see.